packages/main/src/components/AnalyticalTable/types/index.ts

```diff
@@ -56,6 +57,12 @@ export interface ReactTableHooks {
   getResizerProps: any[];
 }
 
+export interface TriggerVertScroll {
```

Comment:
I don't understand the inferface name to be honest? Does is stand for `TriggerVerticalScroll`?

Reply:
Yup, initially I only triggered vertical scroll via state, but as it's now for both direction I'm going to adjust the name.